internal/store/sql_storage.go

```diff
@@ -62,6 +62,44 @@ func (s *sqlStorage) AddComments(ctx context.Context, comments ...*Comment) erro
 	return err
 }
 
+func (s *sqlStorage) QueryUsers(ctx context.Context, opts UserQueryOptions) ([]*User, error) {
+	// TODO Join and Aggregate comments table with users
+	sd := s.dialect.
+		Select(UsersID, UsersName).
+		From(UsersTable)
```

Comment:
goqu is definitely pretty weird feeling to me, I'm not sure exactly what this query looks like with our current SQL structure

I think it's something like outer joining against comments, then aggregating that by like/dislike, then applying the ordering clauses?